var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -18,14 +18,15 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
 
     test_requires_compiler = True
 
-    maintainers("tcojean", "hartwiganzt")
+    maintainers("MarcelKoch", "hartwiganzt")
 
     tags = ["e4s"]
 
-    license("MIT")
+    license("BSD-3-Clause")
```

Comment:
Is the MIT license still valid for older versions?  If so, you can add the `when` argument to indicate when each license applies.

Reply:
The first released license: https://github.com/ginkgo-project/ginkgo/blob/v1.0.0/LICENSE is already BSD-3-clause, so all Ginkgo versions are under this license.